modules/user_data/templates/06_cloudwatch_setup.sh.tpl

```diff
@@ -11,6 +11,11 @@ set -o pipefail
 echo "#################################"
 echo "#    Cloudwatch Provisioning    #"
 echo "#################################"
+
+# Parse the CW Agent Config ffrom SSM store and put it in file
+CWAGENT_CONFIG=$(aws ssm get-parameter --name "/CWAgent/Config" --query "Parameter.Value" --output text)
```

Comment:
Why store the CW agent config in the param store?

Reply:
It is a best practice to host the agent config in the parameter store.